README.md

```diff
@@ -9,156 +9,71 @@ A Flutter package allows you to effortlessly create a animated book widget UI wi
+import 'package:animated_book_widget/animated_book_widget.dart
 ```
 
-3.  Adding AnimatedBookWidget
+## How to use
 
-*With required parameters*
-```dart
-    AnimatedBookWidget(
-      cover: ColoredBox(
-        color: Color(0xFF2D2D2D),
-        child: Image.asset('assets/icons/dvelopers_icon.jpg'),
-      ),
-      content: ColoredBox(color: Color(0xFF0000FF)),
-      size: const Size.fromWidth(160),
-      padding: const EdgeInsets.only(right: 1), // 0(Zero) when you want to set the separation
-    ),
-```    
-
-*With optional parameters*
 ```dart   
-    AnimatedBookWidget(
-      cover: ColoredBox(
-        color: Color(0xFF2D2D2D),
-        child: Image.asset('assets/icons/dvelopers_icon.jpg'),
-      ),
-      content: ColoredBox(color: Color(0xFF0000FF)),
-      size: const Size.fromWidth(160),
-      padding: const EdgeInsets.only(right: 10), // 0(Zero) when you want to set the separation
-      blurRadius: 8,
-      spreadRadius: .1,
-      backgroundBlurOffset: Offset.zero,
-      backgroundColor: color.withOpacity(.5),
-      backgroundShadowColor: color.withOpacity(.075),
-      curve: Curves.linear,
-      animationDuration: Duration(milliseconds: 500),
-      reverseAnimationDuration: Duration(milliseconds: 500)
-    ),
+AnimatedBookWidget(
+  cover: _Cover,
+  content: _Content,
+  size: const Size.fromWidth(160),
+),
 ``` 
 
-*Separation between cards*
+## Params
 
- + Default configuration
-```dart
-    ListView.separated(
-      scrollDirection: Axis.horizontal,
-      clipBehavior: Clip.none,
-      separatorBuilder: (context, index) => const SizedBox(width: 20),
-      itemCount: 3,
-      itemBuilder: (_, index) {
-        final color = Colors.primaries[index % Colors.primaries.length];
-        return AnimatedBookWidget(
-          size: Size.fromWidth(160),
-          cover: ClipRRect(
-            borderRadius: BorderRadius.horizontal(right: Radius.circular(10)),
-            child: ColoredBox(
-              color: Color(0xFF2D2D2D),
-              child: Image.network('https://i.imgur.com/ooKoJzX.jpg'),
-            ),
-          ),
-          content: ColoredBox(
-            color: color,
-            child: Center(
-              child: Text(
-                'D-VELOPERS',
-                style: Theme.of(context)
-                    .textTheme
-                    .displaySmall!
-                    .copyWith(color: Colors.white),
-              ),
-            ),
-          ),
-        );
-      },
-    );
-```    
+```dart   
+AnimatedBookWidget(
+  cover: _Cover,
+  content: _Content,
+  size: const Size.fromWidth(160),
+  padding: const EdgeInsets.only(right: 10),
+  blurRadius: 8,
+  spreadRadius: .1,
+  backgroundBlurOffset: Offset.zero,
+  backgroundColor: color.withOpacity(.5),
+  backgroundShadowColor: color.withOpacity(.075),
+  curve: Curves.linear,
+  animationDuration: Duration(milliseconds: 500),
+  reverseAnimationDuration: Duration(milliseconds: 500)
+),
```

Comment:
when is , or ; ? 

pick one 
 
```
),
```

```
);
```

```
) 
```

Reply:
Explain the reason for the modifications and also indicate which ones were made and why. This should be included in the PR description.